<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrivesWidget</class>
 <widget class="QWidget" name="DrivesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="thisGroupBox">
       <property name="title">
        <string>Hide/Show this drive</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="noChangeThisRadioButton">
          <property name="text">
           <string>No change</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">thisButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="hideThisRadioButton">
          <property name="text">
           <string>Hide this drive</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">thisButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="showThisRadioButton">
          <property name="text">
           <string>Show this drive</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">thisButtonGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Hide/Show all drive</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="noChangeAllRadioButton">
          <property name="text">
           <string>No change</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">allButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="hideAllRadioButton">
          <property name="text">
           <string>Hide all drive</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">allButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="showAllRadioButton">
          <property name="text">
           <string>Show all drive</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">allButtonGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="driveLetterGroupBox">
     <property name="title">
      <string>DriveLetter</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="driveRadioButton">
            <property name="text">
             <string>Existing:</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">driveButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="firstAvailableRadioButton">
            <property name="text">
             <string>Use first available, starting at:</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">driveButtonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QComboBox" name="driveComboBox">
          <property name="font">
           <font>
            <family>Sans Serif</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="currentText">
           <string>A:</string>
          </property>
          <item>
           <property name="text">
            <string>A:</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>B:</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>C:</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>D:</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>E:</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>F:</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>G:</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>H:</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>I:</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>J:</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>K:</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>L:</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>M:</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>N:</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>O:</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>P:</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Q:</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>R:</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>S:</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>T:</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>U:</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>V:</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>W:</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>X:</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Y:</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Z:</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="actionLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>145</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>142</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Action:</string>
       </property>
       <property name="buddy">
        <cstring>actionComboBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="actionComboBox">
       <property name="currentText">
        <string>Create</string>
       </property>
       <item>
        <property name="text">
         <string>Create</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Replace</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Update</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Delete</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="locationFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="pathLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>0</height>
         </size>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string>Location:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="pathLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Sans Serif</family>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QToolButton" name="pathToolButton">
        <property name="font">
         <font>
          <family>Sans Serif</family>
         </font>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="pathLabel_2">
        <property name="text">
         <string>Reconnect:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="reconnectCheckBox">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelLabel">
        <property name="text">
         <string>Label as:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="labelLineEdit">
        <property name="font">
         <font>
          <family>Sans Serif</family>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="thisButtonGroup"/>
  <buttongroup name="allButtonGroup"/>
  <buttongroup name="driveButtonGroup"/>
 </buttongroups>
</ui>
